### ISSUE TITLE
a11y: auro-datepicker accessibility complaint and analysis

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/auro-formkit#497]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [x] Chrome
- [x] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [x] Desktop
- [x] Mobile

### Steps to reproduce

Issue reported in Cherwell ticketing system:
https://cherwell.alaskaair.com/CherwellClient/Access?_=49df2480#0

> "The Alaska Airlines website has instituted a design update that has blocked me from being able to book a flight as a blind screen reader user. I have repeatedly sent in specific accessibility reports and complaints to the airline, but they have not resolved the issue nor fixed the components on the website that are preventing me from using their service. I tried calling to book a flight, but their customer service representatives are prevented from applying account credits when ordering travel. They kept trying to pointlessly redirect me to using their inaccessible website so I could use the credits.
>
>﻿The issues are with the Departure and Arrival airport controls and date picker. I´m unable to set the To and From airports with VoiceOver on MacOS in either Safari or Chrome. The date picker is impossible to use, and ultimately I can never get past this screen to search for flights. I have to resort to sighted assistance just to book travel. This is a violation of the ACAA."

Analyss of the issue and recommendations for improvement are currently being addressed by @blackfalcon 

### Which version of auro-formkit do you have installed?

@latest

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by blackfalcon** [comment]
A full accessibility review and report on the auro-datepicker have been completed. The document has been shared with design and product leadership for review. A meeting will be scheduled to discuss a strategy of remediation.

**Comment by blackfalcon** [comment]
Word document created with findings and initial recommendations. https://alaskaair-my.sharepoint.com/:w:/p/dale_sande/ERu3DV126z5GtpE52IwM2JQBdg0ViIEsUVInQQ-3h6DSxw?e=3V6VT4

Initial meeting with leadership agreed to create ADO story in the appropriate backlog(s) to document the findings to be used once resources have been prioritized to address.

**Comment by jason-capsule42** [comment]
@jason-capsule42 Move all this into our TRD format and then move it to Triage to see if there is any work we want to do based on the TRD.
